src/main/flight/mixer.c

```diff
@@ -721,12 +722,37 @@ static void applyFlipOverAfterCrashModeToMotors(void)
     }
 }
 
+static float applyThrustLinearization(float motorOutput)
+{
+#ifdef USE_THRUST_LINEARIZATION
+    if (motorConfig()->thrustLinearization) {
+        float tl = motorConfig()->thrustLinearization * 0.01f;
+        static float a, a_reci, b, b_sq;
```

Comment:
Betters names would be appreciated.

Reply:
Thing is these are constants in a formula: a and b. and the reciprocal and square of these. I don't see how naming them differently make things better. But open to specific suggestions.

Reply:
Traditionally c programmers like terse code. Though that's not the case at BF. In any case I just removed the b_sq and replaced a by thrustLinerarization during refactoring, so a_reci became thrustLinearization_reci. Please don't make me call it 
```reciprocalOfThrustLinearization = (1.0f / thrustLinearization);```

Reply:
Actually I kind of feel crazy today so I just gave you ``thrustLinearizationReciprocal``.

Reply:
Haha cheers mate!